src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
 }
+
+.header {
+  font-family: Roboto, sans-serif;
+
+  --link-color: #00acdc;
+
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+
+  border: 1px solid white;
+  align-items: center;
+  padding: 0 50px;
```

Comment:
```suggestion
  padding: 0 50px;
  height: 60px;
```

Reply:
Don't understand, what i need to change in padding :(

Reply:
> Don't understand, what i need to change in padding :(